### ISSUE TITLE
[DOC]: total step of gradient accumulation

### ISSUE THREAD

**Issue by binmakeswell** [hpcaitech/ColossalAI#1112]
### 📚 The doc issue

(This issue is collected from user feedback)
When gradient accumulation is turned on, how to set the total step of lr_scheduler with linear warmup, do I still divide it by the accumulation step?

**Comment by binmakeswell** [comment]
Yes, the actual updated step number is used.
